.github/workflows/release.yml

```
@@ -0,0 +1,454 @@
+    if: ${{ !inputs.dry_run && inputs.profile == 'release' }}
+    needs: [preflight, qualify, validate-container]
+    runs-on: ubuntu-24.04
+    timeout-minutes: 120
+    permissions:
+      contents: read
+      id-token: write
+      packages: write
+    outputs:
+      digest: ${{ steps.publish-image.outputs.digest }}
+    env:
+      GITHUB_USER: ${{ secrets.RELEASE_GITHUB_USER || vars.GITHUB_USER || github.actor }}
```

actionlint (1.7.12)

[error] configuration variable name "github_user" must not start with the GITHUB_ prefix (case insensitive). note: see the convention at https://docs.github.com/en/actions/learn-github-actions/variables#naming-conventions-for-configuration-variables

(expression)